src/plugins/kibana_usage_collection/server/collectors/application_usage/rollups.ts

```diff
@@ -0,0 +1,202 @@
+/*
+ * Licensed to Elasticsearch B.V. under one or more contributor
+ * license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright
+ * ownership. Elasticsearch B.V. licenses this file to you under
+ * the Apache License, Version 2.0 (the "License"); you may
+ * not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+import { ISavedObjectsRepository, SavedObject, Logger } from 'kibana/server';
+import moment from 'moment';
+import {
+  ApplicationUsageDaily,
+  ApplicationUsageTotal,
+  ApplicationUsageTransactional,
+  SAVED_OBJECTS_DAILY_TYPE,
+  SAVED_OBJECTS_TOTAL_TYPE,
+  SAVED_OBJECTS_TRANSACTIONAL_TYPE,
+} from './saved_objects_types';
+import { SavedObjectsErrorHelpers } from '../../../../../../src/core/server';
+
+/**
+ * For Rolling the daily data, we only care about the stored the attributes and the version (to avoid overwriting via concurrent requests)
```

Comment:
```suggestion
 * For Rolling the daily data, we only care about the stored attributes and the version (to avoid overwriting via concurrent requests)
```